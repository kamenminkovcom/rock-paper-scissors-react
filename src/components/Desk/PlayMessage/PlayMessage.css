.play-message {
    text-align: center;
    color: #004085;
    background-color: #cce5ff;
    border-color: #b8daff;
    position: relative;
    padding: .75rem 1.25rem;
    margin-bottom: 1rem;
    border: 1px solid transparent;
}