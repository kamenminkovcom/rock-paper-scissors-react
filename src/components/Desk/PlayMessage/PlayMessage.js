import React from 'react';

import './PlayMessage.css';

const PlayMessage = () => (
    <div className="play-message">Please, make your choice.</div>
);

export default PlayMessage;