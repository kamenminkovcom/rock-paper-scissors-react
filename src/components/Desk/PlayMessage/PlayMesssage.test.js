import React from 'react';
import {shallow} from 'enzyme';
import PlayMessage from './PlayMessage';

describe('<PlayMessage/>', () => {
    it('Should render correctly', () => {
        const wrapper = shallow(<PlayMessage/>);
        expect(wrapper).toMatchSnapshot();
    })
});