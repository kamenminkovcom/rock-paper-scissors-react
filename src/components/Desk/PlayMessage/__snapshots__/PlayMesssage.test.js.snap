// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlayMessage/> Should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PlayMessage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Please, make your choice.",
      "className": "play-message",
    },
    "ref": null,
    "rendered": "Please, make your choice.",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Please, make your choice.",
        "className": "play-message",
      },
      "ref": null,
      "rendered": "Please, make your choice.",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
